manifests/02-cncc-credentials.yaml

```diff
@@ -76,3 +76,22 @@ spec:
     kind: AzureProviderSpec
     roleBindings:
       - role: Contributor
+---
+apiVersion: cloudcredential.openshift.io/v1
+kind: CredentialsRequest
```

Comment:
Just FYI for an example:
~~~
[akaris@linux 2105706]$ oc get credentialsrequest -o yaml -n openshift-cloud-credential-operator openshift-openstack-cloud-controller-manager 
apiVersion: cloudcredential.openshift.io/v1
kind: CredentialsRequest
metadata:
  annotations:
    include.release.openshift.io/self-managed-high-availability: "true"
    include.release.openshift.io/single-node-developer: "true"
  creationTimestamp: "2022-07-28T12:20:38Z"
  generation: 1
  name: openshift-openstack-cloud-controller-manager
  namespace: openshift-cloud-credential-operator
  ownerReferences:
  - apiVersion: config.openshift.io/v1
    kind: ClusterVersion
    name: version
    uid: 0e833245-a3db-4b86-bc90-39d82225b45d
  resourceVersion: "3385"
  uid: 5eab667a-ebba-40c9-b78b-771680ebcf59
spec:
  providerSpec:
    apiVersion: cloudcredential.openshift.io/v1
    kind: OpenStackProviderSpec
  secretRef:
    name: openstack-cloud-credentials
    namespace: openshift-cloud-controller-manager
~~~

wrt to the  `include.release.openshift.io/ibm-cloud-managed: "true"` annotation, I added it because the other requests in the same .yaml file also have it

Reply:
I'm new to this repo, so my question isn't necessarily pertinent. I see in this example of yours that the namespace specified under `secretRef` is `openshift-cloud-controller-manager`, while in the yaml in this PR the namespace is `openshift-cloud-network-config-controller`, just like it is for the other CredentialsRequests already in the master branch. Yours was just a custom example, I suppose.

Reply:
Exactly. The cloudcredentials operator installs credentialsrequests in the form of secrets inside the `secretRef`'s target namespace. In the case, I need the secret with openstack cloud credentials inside the openshift-cloud-network-config-controller namespace :-)